lib/node_modules/@stdlib/stats/array/maxsorted/README.md

```diff
@@ -0,0 +1,113 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# maxsorted
+
+> Calculate the maximum value of an sorted array.
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var maxsorted = require( '@stdlib/stats/array/maxsorted' );
+```
+
+#### maxsorted( x )
+
+Computes the maximum value of an sorted array.
+
+```javascript
+var x = [ 1.0, 2.0, 3.0 ];
+
+var v = maxsorted( x );
+// returns 3.0
+
+x = [ 3.0, 2.0, 1.0 ];
+
+v = maxsorted( x );
+// returns 3.0
+```
+
+The function has the following parameters:
+
+-   **x**: input array.
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   If provided an empty array, the function returns `NaN`.
+-   The function supports array-like objects having getter and setter accessors for array element access (e.g., [`@stdlib/array/base/accessor`][@stdlib/array/base/accessor]).
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var maxsorted = require( '@stdlib/stats/array/maxsorted' );
+
+var x = discreteUniform( 10, -50, 50, {
```

Comment:
You need to use `linspace` in order to ensure that `x` is sorted. Otherwise, `x` is an unsorted array which is not what this package is intended to accept.